data_validation/cli_tools.py

```diff
@@ -145,29 +145,37 @@ def configure_arg_parser():
 
     parser.add_argument("--verbose", "-v", action="store_true", help="Verbose logging")
 
-    # beta feature only available in run/validate command
-    if "beta" in sys.argv:
-        parser.add_argument(
-            "beta",
-            nargs="?",
-            help="Beta flag to enable beta features for the tool.",
-            default="",
-        )
-        subparsers = parser.add_subparsers(dest="command")
-        _configure_run_parser(subparsers)
-        _configure_validate_parser(subparsers)
-    else:
-        subparsers = parser.add_subparsers(dest="command")
-        _configure_validate_parser(subparsers)
-        _configure_run_config_parser(subparsers)
-        _configure_connection_parser(subparsers)
-        _configure_find_tables(subparsers)
-        _configure_raw_query(subparsers)
-        _configure_run_parser(subparsers)
+    subparsers = parser.add_subparsers(dest="command")
+    _configure_validate_parser(subparsers)
+    _configure_run_config_parser(subparsers)
+    _configure_connection_parser(subparsers)
+    _configure_find_tables(subparsers)
+    _configure_raw_query(subparsers)
+    _configure_run_parser(subparsers)
+    _configure_beta_parser(subparsers)
 
     return parser
 
 
+def _configure_beta_parser(subparsers):
+    """Configure beta commands for the parser."""
+    connection_parser = subparsers.add_parser(
```

Comment:
The updated beta CLI code looks great. Last thing I would add is documentation in the README for the beta deploy command. Besides that, LGTM